lua/tardis/interiors/legacy.lua

```diff
@@ -50,7 +50,7 @@ T.Interior={
 		rails=true,
 		legacy_throttle={pos=Vector(-8.87,-50,5.5), ang=Angle(-12,-5,24)},
 		legacy_flightlever={pos=Vector(-0.431641, 44.75, 6.4), ang=Angle(-63.913, 137.035, 136.118)},
-		legacy_screen={pos=Vector(42,0.75,27.1), ang=Angle(0,-5,0)},
+		//legacy_screen={pos=Vector(42,0.75,27.1), ang=Angle(0,-5,0)},
```

Comment:
Agree with leaving this as a comment but can you add another comment here explaining why this is commented out?